wee_slack.py

```diff
@@ -1962,6 +1959,12 @@ def get_typing_list(self):
                 del self.typing[user]
         return typing
 
+    def mark_read(self, ts=None, update_remote=True, force=False, post_data={}):
+        if config.thread_messages_in_channel and not update_remote:
+            for thread_channel in self.thread_channels.values():
+                thread_channel.mark_read(ts=ts, update_remote=update_remote, forece=force, post_data=post_data)
```

Comment:
I realized now that this only marks the thread buffers you have opened. If you use `thread_messages_in_channel` the point is that you don't have to open thread buffers. So what we should do is go though `hashed_messages` and for all messages that is a parent message and is unread, we should mark it.

If you want, I can fix this.

Reply:
OK, I removed this for now.  I think that this might be better in another PR, because I think that's also how the other `thread_messages_in_channel` feature would need to be implemented as well.